tools/start_server.py

```diff
@@ -215,13 +221,34 @@ def handle_get_app_manifest_message(params):
 
 	return json.dumps(response_msg)
 
+
+def handle_start_streaming_adapter(params):
+	print("-->Handle start gstreamer adapter\r")
+	stream_endpoint = "http://localhost:8085"
```

Comment:
@LuxoftAKutsan could you please move port number to a separate constant at line 48?

Reply:
bba3698